service/collector/chart_config_resource.go

```diff
@@ -68,8 +68,6 @@ func NewChartConfigResource(config ChartConfigResourceConfig) (*ChartConfigResou
 }
 
 func (c *ChartConfigResource) Collect(ch chan<- prometheus.Metric) error {
-	c.logger.Log("level", "debug", "message", "collecting metrics for ChartConfigs")
```

Comment:
IIRC we added these because wiring went belly up at some point and then it was hard to debug why metrics were missing. Fine with me to remove them if you don't find them useful though.

Reply:
Ack thanks for the explanation. I think we're OK now and I compared with aws-operator which doesn't have these.